### PR TITLE
Tests 1.13

### PR DESCRIPTION
**Pull request by TMcreative1**

### CODE REVIEW

File: src/main/kotlin/com/playwright/remote/engine/route/UrlMatcher.kt [hunk 1]
```diff
@@ -8,7 +8,7 @@ import java.util.regex.Pattern.compile
 
 typealias Predicate = (String) -> Boolean
 
-class UrlMatcher(private val rawSource: Any = "", private val predicate: Predicate = { false }) {
+class UrlMatcher(private val rawSource: Any = "", private val predicate: Predicate? = { false }) {
```
Comment (Roman-Mitusov) [inline src/main/kotlin/com/playwright/remote/engine/route/UrlMatcher.kt:11]: I think if you make Predicate nullable then its default value should be null.

File: src/main/kotlin/com/playwright/remote/engine/route/UrlMatcher.kt [hunk 2]
```diff
@@ -26,6 +26,8 @@ class UrlMatcher(private val rawSource: Any = "", private val predicate: Predica
         }
     }
 
+    constructor() : this("", null)
```
Comment (Roman-Mitusov) [inline src/main/kotlin/com/playwright/remote/engine/route/UrlMatcher.kt:29]: If you will set default value to null for predicate, this default constructor should be just
`constructor() : this("")` as per previous comment.

**Review by Roman-Mitusov — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/playwright/remote/engine/route/UrlMatcher.kt (hunk 1 above)
Comment (TMcreative1) [reply src/main/kotlin/com/playwright/remote/engine/route/UrlMatcher.kt]: Fixed

File: src/main/kotlin/com/playwright/remote/engine/route/UrlMatcher.kt (hunk 2 above)
Comment (TMcreative1) [reply src/main/kotlin/com/playwright/remote/engine/route/UrlMatcher.kt]: Fixed

**Review by Roman-Mitusov — APPROVED** [review]

**Pull request merged**